File: dataset/java2cpp/java/COUNT_WORDS_APPEAR_EXACTLY_TWO_TIMES_ARRAY_WORDS.java
int f_gold ( String str [ ] , int n ) {
  HashMap < String , Integer > m = new HashMap < > ( ) ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( m . containsKey ( str [ i ] ) ) {
      int get = m . get ( str [ i ] ) ;
      m . put ( str [ i ] , get + 1 ) ;
    }
    else {
      m . put ( str [ i ] , 1 ) ;
    }
  }
  int res = 0 ;
  for ( Map . Entry < String , Integer > it : m . entrySet ( ) ) {
    if ( it . getValue ( ) == 2 ) res ++ ;
  }
  return res ;
}

File: result/translate/chatgpt/java2python/python/COUNT_WORDS_APPEAR_EXACTLY_TWO_TIMES_ARRAY_WORDS.py
def f_gold(str, n):
    m = {}
    for i in range(n):
        if str[i] in m:
            get = m[str[i]]
            m[str[i]] = get + 1
        else:
            m[str[i]] = 1
    
    res = 0
    for key, value in m.items():
        if value == 2:
            res += 1
    
    return res